src/external/k8_gcc.BUILD

```diff
@@ -0,0 +1,61 @@
+# -*- python -*-
+# Adapted for linux gcc
+
+# Copyright 2018-2019 Josh Pieper, jjp@pobox.com.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "binaries",
+    srcs = [
+        "usr/bin/x86_64-linux-gnu-gcc",
+        "usr/bin/x86_64-linux-gnu-g++",
+        "usr/bin/x86_64-linux-gnu-ld",
+        "usr/bin/x86_64-linux-gnu-gcc-ar",
+        "usr/bin/x86_64-linux-gnu-gcov",
+        "usr/bin/x86_64-linux-gnu-dwp",
+        "usr/bin/x86_64-linux-gnu-gcc-nm",
+        "usr/bin/x86_64-linux-gnu-gprof",
+        "usr/bin/x86_64-linux-gnu-objcopy",
+        "usr/bin/x86_64-linux-gnu-objdump",
+    ],
+)
+
+filegroup(
+    name = "k8_gcc_libs",
+    srcs = glob(["usr/lib/gcc/x86_64-linux-gnu/7.4.0/*.a"]),
```

Comment:
Similar to my previous comment, should this really all be specifying `7.4.0`?

Reply:
ubuntu 18.04 should have 7.4.0

Reply:
I think the question here is can we make this a bit more general, ie. `usr/lib/gcc/x86_64-linux-gnu/*/*.a` or maybe `usr/lib/gcc/x86_64-linux-gnu/*/*.a`, but I think that could cause issues if there are multiple versions of gcc installed. perhaps we could make this a `filegroup` in the `arm_gcc.BUILD` file, just  so that all the version-specific stuff is in one place?

Reply:
Yes I'm mostly just concerned about us
1. Expecting a specific version of gcc that we do nothing to verify exists
2. "hardcoding" the version in multiple places

I just want this to be easy to upgrade and change in the future, and not cause errors with non-obvious messages if someone without the right setup tries to compile

Reply:
I'm not sure how to filegroup them any better. I think I can reduce changes required to update just `src/external/k8_gcc.BUILD`

Reply:
I think the issue here is maybe more in the naming. Could we rename this file to `linux_gcc_7_4_0`, because that's how specific it is? Similairly, everything in this file should probably be changed from `k8_gcc` to `linux_gcc_7_4_0`. 

This way if we ever go to upgrade the compiler version or even support multiple platforms, it's clear what bits are specific to what.

Reply:
should be good now